### PR TITLE
ZH-trans: add service-accounts-admin.md

### PR DESCRIPTION
**Pull request by xichengliudui**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/reference/access-authn-authz/service-accounts-admin.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+<!--
+### Token Controller
+-->
+
+### 令牌控制器
+
+<!--
+TokenController runs as part of controller-manager. It acts asynchronously. It:
+-->
+TokenController 作为控制器-管理器的一部分运行，异步运行。它:
```
Comment (Mr-Linus) [inline content/zh/docs/reference/access-authn-authz/service-accounts-admin.md]:
```suggestion
TokenController 作为控制器-管理器的一部分异步运行。它用于:
```

File: content/zh/docs/reference/access-authn-authz/service-accounts-admin.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+#### 创建其他API令牌
+
+<!--
+A controller loop ensures a secret with an API token exists for each service
+account. To create additional API tokens for a service account, create a secret
+of type `ServiceAccountToken` with an annotation referencing the service
+account, and the controller will update it with a generated token:
+-->
+控制器循环确保每个 service account 都存在具有 API 令牌的 secret。要为 service account 创建其他 API 令牌，
+请使用 ServiceAccountToken 引用 service account 的注解创建类型的  ，控制器将使用生成的令牌更新它：
```
Comment (Mr-Linus) [inline content/zh/docs/reference/access-authn-authz/service-accounts-admin.md]:
```suggestion
请使用 ServiceAccountToken 引用 service account 的注解创建类型的 secret，控制器将使用生成的令牌更新它：
```

Comment (Mr-Linus) [reply content/zh/docs/reference/access-authn-authz/service-accounts-admin.md]: 这句是不是重复了？

File: content/zh/docs/reference/access-authn-authz/service-accounts-admin.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+  1. If the pod does not contain any `ImagePullSecrets`, then `ImagePullSecrets` of the `ServiceAccount` are added to the pod.
+  1. It adds a `volume` to the pod which contains a token for API access.
+  1. It adds a `volumeSource` to each container of the pod mounted at `/var/run/secrets/kubernetes.io/serviceaccount`.
+-->
+
+  1. 如果 pod 没有设置 `ServiceAccount`，它将 `ServiceAccount` 设置为 `default`。
+  2. 它确保 pod 引用的 `ServiceAccount` 存在，否则拒绝它。
+  3. 如果 pod 不包含任何 `ImagePullSecrets`，则将 `ServiceAccount` 的 `ImagePullSecrets` 添加到 pod 中。
+  4. 它向包含 API 访问令牌的 pod 添加了一个 `volume`。
+  5. 它为安装在 `/var/run/secrets/kubernetes.io/serviceaccount` 上的 pod 的每个容器添加了一个 `volumeSource`。
```
Comment (SataQiu) [inline content/zh/docs/reference/access-authn-authz/service-accounts-admin.md]:
```suggestion
  5. 它为 pod 内的每个容器添加一个 `volumeSource`，该 volume 被挂载到 `/var/run/secrets/kubernetes.io/serviceaccount`。
```

File: content/zh/docs/reference/access-authn-authz/service-accounts-admin.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+-->
+您必须使用 `--service-account-private-key-file` 选项将 service account 私钥文件传递给控制器-管理器中的令牌控制器。
+私钥将用于签署生成的 service account 令牌。同样，您必须使用 `--service-account-key-file` 选项将相应的公钥传递给
+kube-apiserver。公钥将用于在身份验证期间验证令牌。
+
+<!--
+#### To create additional API tokens
+-->
+
+#### 创建其他API令牌
```
Comment (SataQiu) [inline content/zh/docs/reference/access-authn-authz/service-accounts-admin.md]:
```suggestion
#### 创建其他 API 令牌
```

**Pull request merged**